scripts/globals/beastmentreasure.lua

```diff
@@ -0,0 +1,378 @@
+------------------------------------
+require('scripts/globals/keyitems')
+require('scripts/globals/quests')
+require('scripts/globals/status')
+require('scripts/globals/weather')
+------------------------------------
+dsp = dsp or {}
+dsp.beastmentreasure = dsp.beastmentreasure or {}
+
+local zoneData = {
+    -- Yuhtunga Jungle (Elshimo Lowlands)
+    [123] =
+    {
+        statusvar = 'BMT_Lowlands_Status',
+        qmvar = 'BMT_Lowlands_DigQM',
+        mapid = dsp.ki.MAP_OF_THE_ELSHIMO_REGIONS,
+        peddlestox =
+        {
+            id = 17281639,
+            day = dsp.day.WINDSDAY
+        },
+        fetchitems =
+        {
+            1480, -- Mermaid Head
+            1481, -- Mermaid Body
+            1482, -- Mermaid Hands
+            1483  -- Mermaid Tail
+        },
+        digsites =
+        {
+            [3]  = {id = 17281642, x = 83.726,   y = 448.329,  chest = 0},
+            [4]  = {id = 17281643, x = -114.254, y = -125.471, chest = 1},
+            [5]  = {id = 17281644, x = -300.823, y = 9.208,    chest = 2},
+            [6]  = {id = 17281645, x = -539.311, y = 203.151,  chest = 3},
+            [7]  = {id = 17281646, x = -577.734, y = -82.563,  chest = 4},
+            [8]  = {id = 17281647, x = -361.835, y = -399.517, chest = 5},
+            [9]  = {id = 17281648, x = -294.827, y = -453.237, chest = 6},
+            [10] = {id = 17281649, x = 369.795,  y = 201.805,  chest = 7}
+        },
+        loot =
+        {
+            unique = {[14626] = 5}, -- Mermaid's Ring
+            racial = {[887] = 3}    -- Coral Fragment
+        }
+    },
+    -- Yhoator Jungle (Elshimo Uplands)
+    [124] =
+    {
+        statusvar = 'BMT_Uplands_Status',
+        qmvar = 'BMT_Uplands_DigQM',
+        mapid = dsp.ki.MAP_OF_THE_ELSHIMO_REGIONS,
+        peddlestox =
+        {
+            id = 17285685,
+            day = dsp.day.LIGHTNINGDAY
+        },
+        fetchitems =
+        {
+            1484, -- Rancor Mantle
+            1485, -- Rancor Globe
+            1486, -- Rancor Tank
+            1487  -- Rancor Handle
+        },
+        digsites =
+        {
+            [5]  = {id = 17285688, x = 80.52,    y = 200.147,  chest = 1},
+            [6]  = {id = 17285689, x = 198.65,   y = 58.046,   chest = 2},
+            [7]  = {id = 17285690, x = 205.932,  y = -174.886, chest = 3},
+            [8]  = {id = 17285691, x = 525.239,  y = -394.046, chest = 4},
+            [9]  = {id = 17285692, x = 240.341,  y = -402.532, chest = 5},
+            [10] = {id = 17285693, x = 16.069,   y = -393.843, chest = 6},
+            [11] = {id = 17285694, x = -255.358, y = -405.178, chest = 7},
+            [4]  = {id = 17285695, x = -149.929, y = 56.761,   chest = 0}
+        },
+        loot =
+        {
+            unique = {[13400] = 5}, -- Bitter Earring
+            racial = {[4158] = 3}   -- Venom Potion
+        }
+    },
+    -- Western Altepa Desert (Kuzotz)
+    [125] =
```

Comment:
```lua
[dsp.zone.WESTERN_ALTEPA_DESERT]
```

Reply:
Resolved in https://github.com/DarkstarProject/darkstar/pull/6095/commits/d3e56c44ade40cbcb41fe95e8b7ba65acc37f883